addons/point_of_sale/models/report_sale_details.py

```diff
@@ -164,7 +164,7 @@ def get_sale_details(self, date_start=False, date_stop=False, config_ids=False,
                         elif payment['id'] in account_payments.mapped('pos_payment_method_id.id'):
                             account_payment = account_payments.filtered(lambda p: p.pos_payment_method_id.id == payment['id'])
                             payment['final_count'] = payment['total']
-                            payment['money_counted'] = sum(account_payment.mapped('amount'))
+                            payment['money_counted'] = sum(account_payment.mapped('amount_signed'))
```

Comment:
`amount_signed` seems to be a negative amount , is it the goal ? 

```py
    amount_signed = fields.Monetary(
        currency_field='currency_id', compute='_compute_amount_signed', tracking=True,
        help='Negative value of amount field if payment_type is outbound')
```

Reply:
@davidmonnom yes